src/functions/keycloak/remove-user-attribute/handler.ts

```diff
@@ -0,0 +1,41 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import { CreateKeycloakApiService } from "src/database/services/keycloak-api-service";
+import { middyfy } from "src/libs/lambda";
+
+/**
+ * Lambda handler to remove a user's attribute
+ * 
+ * @param event API Gateway event
+ * @returns Response message as JSON string
+ */
+const removeUserAttributeHandler: APIGatewayProxyHandler = async (
+  event: APIGatewayProxyEvent
+) => {
+  try {
+    const id = event.pathParameters?.id;
+    const attributeName = event.pathParameters?.attributeName;
+
+    if (!id || !attributeName) {
+      return {
+        statusCode: 400,
+        body: JSON.stringify({ message: "Missing required path parameters: 'id' or 'attributeName'." }),
+      };
+    }
+
+    const api = CreateKeycloakApiService();
+
+    await api.removeUserAttribute(id, attributeName);
+
+    return {
+      statusCode: 200,
+      body: JSON.stringify({ message: `User attribute "${attributeName}" removed successfully.` }),
```

Comment:
If we are treating this as a delete endpoint it should return the appropriate status and body e,g,
   return {
      statusCode: 204,
      body: "",
    };